templates/server/index.html

```diff
@@ -1,460 +1,484 @@
-          </li>
-          <li class="p-inline-images__item">
-            <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/683950fd-logo-ibm.svg" width="144" alt="IBM" />
-          </li>
-          <li class="p-inline-images__item">
-            <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/a298e7ec-logo-hp.svg" width="88" alt="HP" />
-          </li>
-          <li class="last-item p-inline-images__item">
-            <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/c24cb4b9-logo-intel.svg" width="132" alt="Intel" />
-          </li>
-          <li class="p-inline-images__item">
-            <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/4d6054f9-logo-cisco.svg" width="144" alt="Cisco" />
-          </li>
-          <li class="p-inline-images__item">
-            <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/731aab6c-logo-amd.svg" width="144" alt="AMD" />
-          </li>
-          <li class="last-item p-inline-images__item">
-            <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/4efbd291-Arm_logo_blue_cropped.svg" width="144" alt="Arm" />
-          </li>
-        </ul>
-        <p class="p-link--external u-align--center u-vertically-spaced"><a href="https://certification.ubuntu.com/" class="p-link--external">查看所有认证硬件</a></p>
 
-        <ul class="p-inline-images">
-          <li class="p-inline-images__item">
-            <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/76999dfc-logo-centrify.png?w=132" width="132" alt="Centrify" />
-          </li>
-          <li class="p-inline-images__item">
-            <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/eed716e9-logo-openstack.svg" width="88" alt="OpenStack" />
-          </li>
-          <li class="p-inline-images__item">
-            <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/c228b4e9-logo-likewise.png?w=144" width="144" alt="Likewise" />
-          </li>
-          <li class="last-item p-inline-images__item">
-            <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/64493bba-logo-openbravo.png?w=144" width="144" alt="Openbravo" />
-          </li>
-        </ul>
-        <p class="u-align--center u-vertically-spaced"><a class="p-link--external" href="https://partners.ubuntu.com/programmes/software">查看所有认证软件</a></p>
-      </div>
+    <div class="u-fixed-width">
+      <p class="p-heading--four">定期更新和升级确保系统处于最新状态</p>
+      <p>Canonical为Ubuntu Server长期支持版（LTS）提供默认五年的安全更新支持。每隔半年，中间版本会带来新的功能，硬件支持更新会在所有受支持的LTS版本中增加对最新硬件的支持。所有<a href="https://www.ubuntu.com/support">Ubuntu Advantage for Infrastructure订阅服务</a>已包含<a href="https://ubuntu.com/esm">扩展安全维护——Extended Security Maintenance (ESM)</a>使得支持周期最多延长至10年。</p>
     </div>
+
+<div class="row">
```

Comment:
This chart shouldn't be hard-coded, it's generated with a library on ubuntu.com so that it's easier to maintain

https://github.com/canonical-web-and-design/ubuntu.com/blob/master/templates/shared/_release_schedule.html